scripts/img-copy.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+# This script is a helper meant for copying images to your own account.
+# Currently it supports AWS only and requires the below parameters:
+#
+#  1 AWS Credentials profile name
+#  2 AMI ID
+#  3 target regions, commma separated
+#
+# To have the proper AWS credentials, you'll need to use gimme-aws-creds along with
+# Okta - this is the company standard, but technically nothing prevents you
+# from setting up a custom ~/.aws/credentials file. Contents the credentials file
+# will also provide you the necessary value for the AWS credentials profile name.
+# Having Docker installed on your machine is also required to run this script.
+#
+# Example use:
+# ./img-copy.sh default ami-0073e5fb98e2faa3f eu-north-1,eu-west-3
+#
+# This will cause you to end up with a whole lot of log on the standard out, ending
+# with something like this:
+# + echo '2023-11-23 11:00:32+00:00 Image copied to regions: eu-north-1=ami-0b7149651a90989f3,eu-west-3=ami-0fb5aa19986c12ec8'
+# Now you can copy those AMI references into a custom catalog.
+#
+# If you run into the below error message when the script finishes, it means that 
+# your account can't have public AMIs:
+#   "An error occurred (OperationNotPermitted) when calling the ModifyImageAttribute 
+#    operation: You can’t publicly share this image because block public access for
+#    AMIs is enabled for this account. To publicly share the image, you must call 
+#    the DisableImageBlockPublicAccess API."
+# Having public AMIs is a must though, as Cloudbreak needs to access them, so you'll 
+# have to enable this in the account.
+#
+# Note: Regardless of the above error message, the AMI copy should be successful.
+
+if [[ -z $1 ]]; then
+  echo AWS Credentials profile must be provided as the second parameter!
+  exit -1
```

Comment:
As far as I know exit code must be in the 0 - 255 range. Am I wrong? -1 is going to be mapped to 255 so if you really need 255 then plese use 255.